fma/queries.py

```diff
@@ -0,0 +1,71 @@
+import purl
+import requests
+from django.conf import settings
+
+from . import models
+
+API_KEY = getattr(settings, 'FMA_API_KEY', None)
+
+
+class FMAQueryException(Exception):
+    """Base exception for errors querying the FMA API"""
+
+
+class FMAServerError(FMAQueryException):
+    """Raised when an error status code is found on a response from the FMA server"""
+
+
+class FMAQueryError(FMAQueryException):
+    """Error indicated in the API response, probably an error in the request"""
+
+
+def query_tracks(artist=None):
+    track_resource = "http://freemusicarchive.org/api/get/tracks.json"
+    query_url = purl.URL(track_resource).query_params({
+        'api_key': API_KEY,
+        'artist_id': artist.pk,
+    }).as_string()
+    response = requests.get(query_url)
+    if response.status_code >= 500:
+        raise FMAServerError("Failed to query FMA artist due to server error.")
+    track_data = response.json()
+    if track_data['errors']:
+        raise FMAQueryError("Error querying Free Music Archive: {}".format(track_data['errors']))
```

Comment:
@treyhunner, it seems artists have been removed from the API. Many that we know of already are not present when I query this track endpoint. Do you want to just silently flag them as inactive or removed? Remove them from our data all together?